src/components/breadcrumbs/breadcrumbs-test.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Breadcrumbs } from ".";
+import { Crumb } from "./crumb";
+
+export default {
+  title: "Breadcrumbs/Test",
+  includeStories: "DefaultCrumb",
+  parameters: {
+    info: { disable: true },
+    chromatic: {
+      disableSnapshot: true,
+    },
+  },
+  argTypes: {
+    isCurrent: {
+      control: {
+        type: "boolean",
+      },
+    },
+    href: {
+      control: {
+        type: "text",
+      },
+    },
+  },
+};
+
+export const Default = ({ ...args }) => {
+  return (
+    <Breadcrumbs {...args}>
+      <Crumb href="#">Breadcrumb 1</Crumb>
+      <Crumb href="#">Breadcrumb 2</Crumb>
+      <Crumb href="#">Breadcrumb 3</Crumb>
+      <Crumb href="#" isCurrent>
+        Current Page
+      </Crumb>
+    </Breadcrumbs>
+  );
+};
+
+export const DefaultCrumb = ({ ...args }) => {
```

Comment:
Sorry I've just noticed this, is there any point in displaying a single crumb? The story looks pretty weird and I don't think we'll ever have it like this in production? I think this story can just be deleted and `includeStories: "DefaultCrumb",` can be updated to `includeStories: "Default",`


![image](https://user-images.githubusercontent.com/44157880/228783703-6e73525e-f81d-4514-9a16-20a8f4c3d37c.png)

Reply:
this is a `TEST` story :)